Error: An interface property cannot have an initializer

The published TypeScript definitions seem to contain an error:

```
../../../node_modules/@blocto/sdk/dist/blocto-sdk.d.ts:17:25 - error TS1246: An interface property cannot have an initializer.

17   code: string | null = null;
```

<img width="649" alt="Screen Shot 2021-12-12 at 5 21 10 PM" src="https://user-images.githubusercontent.com/1173161/145733738-c3bff58d-ce6d-4e77-8ae4-d2abdb7ab916.png">

This is using `"@blocto/sdk": "^0.2.20"`.

--- a/src/providers/types/blocto.d.ts
+++ b/src/providers/types/blocto.d.ts
@@ -8,7 +8,7 @@ declare interface BloctoProviderInterface extends EIP1193Provider {
   eventListeners: {
     [key: string]: Array<(arg?: any) => void>
   };
-  code: string | null = null;
+  code: string | null;
   sessionKey: string;
 }
 

